index.bs

```diff
@@ -4848,6 +4848,13 @@ Switching Tracks</h3>
 	their new track’s initial maturity stage,
 	while retaining any established identity (url, shortname, etc.).
 
+	Note: The initial maturity stage of the [=Recommendation track=] is [=Working Draft=].
+	[=First Public Working Draft=] designates a specific type of [=Working Draft=]
+	and is not a separate maturity stage.
+	A document which switches to the [=Recommendation track=] is only published as a [=First Public Working Draft=]
+	if it was never previously published as such,
+	otherwise it is simply a [=Working Draft=].
```

Comment:
Sorry for the timing; I was on holiday.
```suggestion
	if it was never previously published as such;
	otherwise, it is simply a [=Working Draft=].
```

Reply:
Thanks. Suggestion adopted.